dashboard/src/app/workspaces/create-workspace/project-source-selector/add-import-project/template-selector/template-selector-item/template-selector-item.html

```diff
@@ -12,15 +12,16 @@
            class="che-checkbox-area template-selector-item-checkbox">
         <che-list-item-checked ng-model="templateIsChecked[template.name]"
                                che-aria-label-checkbox="Sample {{template.name}}"
+                               id="sample-{{template.name}}"
                                ng-click="onTemplateClick()"></che-list-item-checked>
       </div>
 
       <div flex="25" layout="column" layout-align="center start">
-        <span class="che-list-item-name template-selector-item-name">{{template.name}}</span>
+        <span class="che-list-item-name template-selector-item-name" template-name="{{template.name}}">{{template.name}}</span>
       </div>
 
       <div flex layout="column" layout-align="center start">
-        <span class="che-list-item-secondary template-selector-item-description">{{template.description}}</span>
+        <span class="che-list-item-secondary template-selector-item-description" template-description="{{template.description}}">{{template.description}}</span>
```

Comment:
_template-description_ attribute looks redundant so as we can get *"template-description"* value from content of span.